modules/gapi/test/own/mat_tests.cpp

```diff
@@ -384,4 +384,32 @@ TEST(OwnMat, ROIView)
     << to_ocv(roi_view) << std::endl
     << expected_cv_mat  << std::endl;
 }
+
+TEST(OwnMat, CreateNegEmptyMat)
+{
+    // We use create method for empty own::Mat
+    Mat own_mat_t1, own_mat_t2, own_mat_t3;
+    EXPECT_THROW(own_mat_t1.create(cv::Size{-1, -1}, CV_8U), std::exception);
```

Comment:
std::exception is base class it coverage all exceptions, better to use `EXPECT_ANY_THROW` if you don't want to clarify exception type

Reply:
Fixed.